src/Stratis.Bitcoin.Features.RPC.Tests/Controller/RPCControllerTest.cs

```diff
@@ -46,14 +46,13 @@ public RPCControllerTest()
             this.fullNode.Setup(f => f.Network)
                 .Returns(this.testNetwork);
             this.rpcHost = new Mock<IWebHost>();
-            var nodeSettings = new NodeSettings(this.testNetwork, args: new string[] { "-server=1" });
+            var nodeSettings = new NodeSettings(this.testNetwork, args: new string[] { "-server=1", "-rpcuser=test", "-rpcpassword=test", "-rpcport=0" });
             this.rpcSettings = new RpcSettings(nodeSettings);
             this.serviceProvider = new Mock<IServiceProvider>();
             this.rpcClientFactory = new Mock<IRPCClientFactory>();
             this.actionDescriptorCollectionProvider = new Mock<IActionDescriptorCollectionProvider>();
 
             this.rpcClient = new Mock<IRPCClient>();
-            this.rpcSettings.Bind.Add(new IPEndPoint(IPAddress.Parse("127.0.0.1"), 0));
```

Comment:
This is now not required?

Reply:
Yes the defaults are now being established even if server=0 is specified. This allows us to simply set server=1 at a later stage if the app wants to do it that way.